client/src/App.js

```diff
@@ -25,6 +25,7 @@ function App() {
     })
   }
   function handleLogin() {
+    const user_name = document.getElementById("user_name").value
```

Comment:
Not sure if this is the ideal way, but it prevents passing another handler to change user_name. Makes code cleaner.